fluminum/app/(tabs)/spirit-level.tsx

```diff
@@ -18,12 +19,25 @@ export function calculatePoints(topBottomDeg: number, leftRightDeg: number) {
   const pointsLeft = middlePoint + ` ${leftX},${leftY1} ${leftX},${leftY2}`;
   return { pointsTop, pointsLeft };
 }
+
+export function calculateCorrectAngleUnit(angleUnit: string, radValue: number) {
```

Comment:
you could do something like this

```ts
type AngleUnit = "rad" | "deg" | "percent"
```

And then use that everywhere instead of just `string` :)